samples/matter/thermostat/src/temperature_manager.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "app_event.h"
+
+#include <app-common/zap-generated/attributes/Accessors.h>
+
+#include <lib/core/CHIPError.h>
+
+using namespace chip;
+
+// AppCluster Spec Table 85.
+enum
+{
+    OFF = 0,
+    AUTO,
+    NOT_USED,
+    COOL,
+    HEAT,
+}SystemMode;
+
+
+static uint8_t sReturnCompleteValue(int16_t Value);
+static uint8_t sReturnReminderValue(int16_t Value);
```

Comment:
I guess you tried to declare ReturnReminderValue & ReturnCompleteValue functions here, but `s` has gone away. Moreover, I guess we should stick C++ style in our samples. Follow up on that in the comment regarding the usage of these functions.

Reply:
That might be misunderstood of previous comment of @markaj-nordic 
![image](https://github.com/LipinskiPNordicSemi/sdk-nrf/assets/132913157/5c45d8dc-1f8c-4ef7-b30f-6af2865e5783)